exercises/Loops-basic-vector-R.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: exercise
+topic: Loops
+title: Basic Vector
+language: R
+---
+
+The following code capitalizes the first letter of `pet` and stores it as
+`name`:
+
+```
+library(stringr)
+pet <- "spot"
+pet_letters <- str_split(pet, "")[[1]]
+first <- toupper(pet_letters[1])
+others <- str_c(pet_letters[-1], collapse = "")
+name <- str_c(first, others, sep = "")
+```
```

Comment:
A simpler version of this would be:

```
library(stringr)
pet <- "spot"
name <- str_to_title(pet)
```

The problem is that since `stringr` is vectorized then this solves the overall problem:

```
library(stringr)
pets <- c("spot", "gigantor", "fluffy")
names <- str_to_title(pet)
```

which is maybe why you took the more complicated approach.